compiler/options.nim

```diff
@@ -587,6 +588,17 @@ proc pathSubs*(conf: ConfigRef; p, config: string): string =
   if "~/" in result:
     result = result.replace("~/", home & '/')
 
+proc nimbleSubs*(conf: ConfigRef; p: string): seq[string] =
+  let pl = p.toLowerAscii
+  if "$nimblepath" in pl or "$nimbledir" in pl:
```

Comment:
I put both because Nimble calls this `nimbleDir` whereas Nim calls it `nimblePath` in its command line help as well as in the code.

Further, there's `$projectdir` and `$projectpath` in `pathSubs()` which are synonyms so I added the same for `$nimble*` as well.